gobblin-service/src/main/java/org/apache/gobblin/service/modules/utils/FlowCompilationValidationHelper.java

```diff
@@ -64,10 +64,12 @@ public final class FlowCompilationValidationHelper {
    * For a given a flowSpec, verifies that an execution is allowed (in case there is an ongoing execution) and the
    * flowspec can be compiled. If the pre-conditions hold, then a JobExecutionPlan is constructed and returned to the
    * caller.
+   * @param flowSpec
+   * @param optionalFlowExecutionId provided for executions of scheduled events which should use a consistent id
```

Comment:
how about
```
for scheduled (non-ad-hoc) flows, to pass the ID "laundered" via the DB; see: {@link MultiActiveLeaseArbiter#whateverMethodHasJavadocToDescribe}
```
?

i.e. there should already be good docs about this, so merely help the maintainer find them w/ a link